tests/functional/spec/components/radio/radio.spec.js

```diff
@@ -0,0 +1,161 @@
+const helpers = require('../../../helpers');
```

Comment:
Can we move radio.spec.js into the components folder and remove the radio folder?